android/modules/ui/src/java/ti/modules/titanium/ui/TableViewProxy.java

```diff
@@ -137,6 +139,13 @@ public void releaseViews()
 	@Override
 	public void release()
 	{
+		// Release search bar proxy if there is one
+		if (getTableView().getNativeView() instanceof RelativeLayout) {
+			((RelativeLayout) getTableView().getNativeView()).removeAllViews();
+			TiViewProxy searchView = (TiViewProxy) (getProperty(TiC.PROPERTY_SEARCH));
```

Comment:
no need to wrap getProperty in parens.
```suggestion
			TiViewProxy searchView = (TiViewProxy) getProperty(TiC.PROPERTY_SEARCH);
```